### PR TITLE
Feat/face payment

### PR DESCRIPTION
**Pull request by rizkybaihaqy**
## Type of changes

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Enhancement
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behaviour?

face payment is currently stuck in the checkout process

## What is the new behavior?

Add face payment pay feature with phone number, face match and liveness and pin if exceed the maximum threshold.

- create big input component
- handle check-limit api and pay api

## Checklist

- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have filled the `[Unreleased]` [Changelog](https://github.com/nodefluxio/weber/blob/main/CHANGELOG.md

## Security Pentester API Checklist
- [ ] IDOR
- [ ] Malicious Script Injection
- [ ] SQL Injection

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

not tested with backend

### CODE REVIEW

File: frontend/app/components/modules/PaymentPay/PaymentPay.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        </div>
+      )}
+      {step === 3 && (
+        <div className={styles.pinInputWrapper}>
+          <PinInput
+            message={'You’ve reach your minimum payment, please input your pin'}
+            digits={PIN_DIGIT_LENGTH}
+            onPinChange={setPinCode}
+            isDisabled={isPinCreated}
+          />
```
Comment (williammfu) [inline frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: Kalo salah pin --> try again --> balik ke pin input.
Fieldnya masih ke disabled jadi gabisa ngisi

Comment (rizkybaihaqy) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: sebeernya gw lumayan bingung da fu, ini tujuan awal di disable apa ya?

Comment (williammfu) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: hmm, buat aktivasi juga gabutuh sih skrg

Comment (williammfu) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: di-omit aja kali ya isDisabled nya

Comment (rizkybaihaqy) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: okok fu

Comment (rizkybaihaqy) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: dari komponenya udh required fu. gw buat opsioal aja ya

Comment (williammfu) [reply frontend/app/components/modules/PaymentPay/PaymentPay.tsx:101]: gue baru pull req ky yg revisi pin field nya

**Review by gregoriusjimmy — APPROVED** [review]

**Review by williammfu — APPROVED** [review]

File: frontend/app/api/paymentAPI.ts [hunk 2]
```diff
@@ -86,6 +88,67 @@ export const activateAccount = async (
...
 
+export const checkLimit = async (
+  sessionId: string,
+  phone: string,
+  amount: number
+): Promise<CheckLimitResponse | undefined> => {
+  try {
+    const res = await axios.post<CheckLimitResponse>(
+      `/face-payment/check-limit`,
+      {
```
Comment (krismp) [inline frontend/app/api/paymentAPI.ts:99]:
```
{
        session_id: sessionId,
        phone,
        amount
 }
```

Kalo `key` sama `value` nya sama bisa gini

**Pull request merged**